2-mandatory/3-stocks.js

```diff
@@ -47,10 +61,28 @@ function getAveragePrices(closingPricesForAllStocks) {
                 (Apple's price on the 5th day) - (Apple's price on the 1st day) = 172.99 - 179.19 = -6.2
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
-function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
+
+
+function getPriceChanges(array) {
+
+    let changeInPrice = [ ];
+
+    for (i = 0; i < array.length; i++) {
+        let closingPricesArray = array;
+        let lengthOfInnerArray = closingPricesArray[i].length;
+        let fifthDayPrice = closingPricesArray[i][lengthOfInnerArray - 1];
+        let firstDayPrice = closingPricesArray[i][0];
+
+        let fifthDayMinusFirstDayRounded = Math.round((fifthDayPrice - firstDayPrice) * 100) / 100;
```

Comment:
Suggestion do `* 100` on `fifthDayPrice` and `firstDayPrice` separately so the `-` operation will occur on integers instead of floating point numbers, reducing floating point math inaccuracy.